packages/webapp/pages/onboarding.tsx

```diff
@@ -62,22 +62,30 @@ import { defaultOpenGraph, defaultSeo } from '../next-seo';
 const ContentTypes = dynamic(() =>
   import(
     /* webpackChunkName: "contentTypes" */ '@dailydotdev/shared/src/components/onboarding/ContentTypes/ContentTypes'
-  ).then((mod) => mod.ContentTypes),
+  ).then((mod) => ({
+    default: mod.ContentTypes,
+  })),
```

Comment:
Why is this needed?

Reply:
6a090669e21b4303/node_modules/@next/codemod/transforms/next-dynamic-access-named-export.js

Reply:
I am not sure we want this, it feels redundant?

Reply:
Reverted, guess it's fine :)

Reply:
I am fine with leaving it, was just curious why was it there 👍